modules/specifying-the-environment-variables-in-the-securedcluster-cr.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * cloud_service/installing_cloud_ocp/configuring-the-proxy-for-collector-in-rhacs-cloud-service.adoc
+:_content-type: PROCEDURE
+[id="specifying-the-environment-variables-in-the-securedcluster-cr_{context}"]
+= Specifying the environment variables in the SecuredCluster CR
+
+[role="_abstract"]
+To configure an egress proxy, you can either use the cluster-wide {osp} proxy or specify the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables within the SecuredCluster Custom Resource (SecuredCluster CR) configuration file to ensure proper use of the proxy and bypass for internal requests within the specified domain.
```

Comment:
I think you can just have `CR` bc it the only thing that needs the abbreviation is CR 

```suggestion
To configure an egress proxy, you can either use the cluster-wide {osp} proxy or specify the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables within the SecuredCluster Custom Resource (CR) configuration file to ensure proper use of the proxy and bypass for internal requests within the specified domain.
```